includes/Core/Assets/Assets.php

```diff
@@ -212,10 +226,7 @@ public function enqueue_fonts() {
 
 		$this->fonts_enqueued = true;
```

Comment:
This property should also not be set until after we know for sure that SK will be enqueuing the fonts – ie. after we've applied the filter and ensured it's not empty.